src/main/resources/swagger.api/schema/resource/WorkReference.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
Do we really want to create this `WorkReference` class @PBobylev?
We had this class earlier. Later we removed it as the content of this class is exactly same as the Work class. Having 2 classes forced us to add the following code in all MapperUnits

```
else if (parentDto instanceof WorkReference workReference) {
      workReference.addXYZ(....);
}
```

I see that the only difference between Work and WorkReference is  the presence of this line in Work -> `"x-field-extra-annotation": "@NotEmpty"`. It should be safe to assume that a Work (and Instance) will always have a title. So, I do not see any problem in adding @NotEmpty validation to the Work class and using  this class for both request and response.

Reply:
I also didn't want to return it. The reason is: we can't do the same validation for Instance/Work Reference DTO, because such DTO in a request contains only ID as a reference. And response will contain all the data.
Probably we should distinguish our DTOs between "request/response" directions and not "reference/not reference"

Reply:
Got it. Distinguishing DTOs for request & response is a good idea.

Reply:
Doing it in a new PR, because it's too big fir this